pom.xml

```diff
@@ -127,7 +127,7 @@
 
     <!-- required to fix RHDM-293 -->
     <version.org.apache.logging.log4j.log4j-to-slf4j>2.9.0</version.org.apache.logging.log4j.log4j-to-slf4j>
-
+    
```

Comment:
please remove just format changes from this PR

Reply:
Fixed as per suggested